core/src/allocation_optim.jl

```diff
@@ -125,19 +78,15 @@ function set_objective_priority!(
     t::Float64,
```

Comment:
This function will also be moved to `allocation_init.jl` (just to let you know).